src/lib/src/select-container.component.ts

```diff
@@ -137,7 +131,7 @@ export class SelectContainerComponent implements AfterViewInit, OnDestroy {
 
       const show$ = dragging$.pipe(mapTo(1));
       const hide$ = mouseup$.pipe(mapTo(0));
-      const opacity$ = merge(show$, hide$, asap).pipe(distinctUntilChanged());
+      const opacity$ = merge(show$, hide$, animationFrameScheduler).pipe(distinctUntilChanged());
```

Comment:
the `asapScheduler` gave run time exceptions, changed it to `animationFrameScheduler` for now. 
Gave it a quick look but couldn't fix it right away, will take a look in the next couple of days.

Reply:
This scheduler can be removed anyways.

Reply:
Ok, I'll remove it then.